Packs/CommonPlaybooks/TestPlaybooks/playbook-Eradication_plan_-_Test.yml

```diff
@@ -0,0 +1,640 @@
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 850
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "9":
+    id: "9"
+    taskid: 9fa8f623-1b36-4610-893a-a6463414a5dc
+    type: condition
+    task:
+      id: 9fa8f623-1b36-4610-893a-a6463414a5dc
+      version: -1
+      name: Verify eradication action succeed
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "11"
+      "yes":
+      - "10"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: TerminatedProcessFromEndpoints
+            iscontext: true
+          right:
+            value: {}
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1210
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "10":
+    id: "10"
+    taskid: 104cf899-ac2a-4bf5-815b-732e28101bba
+    type: title
+    task:
+      id: 104cf899-ac2a-4bf5-815b-732e28101bba
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1580
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "11":
+    id: "11"
+    taskid: 5507444c-048d-4259-8aa1-85661696a565
+    type: regular
+    task:
+      id: 5507444c-048d-4259-8aa1-85661696a565
+      version: -1
+      name: Make test fail
+      description: Prints an error entry with a given message
+      scriptName: PrintErrorEntry
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "10"
+    scriptarguments:
+      message:
+        simple: The eradication plan playbook has failed. Please check the output of the eradication sub-playbooks.
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 740,
+          "y": 1400
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "15":
+    id: "15"
+    taskid: 93c4740c-74b3-42e2-8e37-3d8630728eeb
+    type: playbook
+    task:
+      id: 93c4740c-74b3-42e2-8e37-3d8630728eeb
+      version: -1
+      name: Eradication Plan
+      playbookName: Eradication Plan
```

Comment:
I see that the default option for remediation is `delete` - and you are planning to delete the notepad.exe file
r u sure about that?